### ISSUE TITLE
Failed to load the Schematic

### ISSUE THREAD

**Issue by joveaaron** [maruohon/litematica#307]
**Minecraft version**
1.16.5

**Mod version and malilib version**
litematica-fabric-1.16.4-0.0.0-dev20210120.213004.jar
malilib-fabric-1.16.4.0.10.0-dev.21+arne.2.jar

**Description of the bug**
"Failed to load the Schematic" error while trying to load a Schematic from a 1.16.5 WorldEdit Schematic.

Hello. I was trying to make an automatic nether portal that I created in creative and now I want it in my survival world. When loading the Schematic that error happened and now I don't know what to do.

I found a thread that specifically said that it was because of a thing of 1.12 or something but in the end, you said that it was fixed on a version with a date.

Are there any fixes to this?
[Schematic](http://www.mediafire.com/file/nuhlmx6t8x0332c/nether_portal_redstone_automatic.schematic/file) file.

**Comment by maruohon** [comment]
Sponge schematic format (which is used by World Edit for example) is not supported in the current 1.13.2+ versions of Litematica yet. So you will either have to paste it to a temporary world with World Edit an then save it from that with Litematica, or just save it with Litematica directly rom the original build, if you even created it.

The difference is that if there are any items in inventories that you would want the schematic to have (for example so that you can use the Block Info Overlay to see how many items are needed in each inventory), then you can't save it with Litematica from a server, as the server does not sync the inventory contents to the client normally, except when you open the inventory GUI.

So basically if you created this on a creative server, and you want the inventory contents in the schematic, then use World Edit to save it from the server, then paste it to a temporary single player world with World Edit, and save it from that with Litematica. If you made it in single player in the first place, or you don't care about inventory contents, then you can just directly save it with Litematica from the original build.

**Comment by joveaaron** [comment]
Ok thank you. I got it to work loading it into a flat world.
**Closing thread.**
